content/components/dialog.mdx

```diff
@@ -255,6 +255,12 @@ If the answer is yes, designing the feature inside a dialog provides an easy way
   </Dont>
 </DoDontContainer>
 
+### Nested dialogs
+
+It is acceptable to have up to two nested dialogs as long as the dialog is implemented correctly. For example, upon closing the second nested dialog, focus should return to the trigger control for this dialog within the first dialog. When the two stacked dialogs are open, pressing `Esc` should only close the top-level dialog.
```

Comment:
Add a note here about clicking outside the second dialog only closing the second dialog as well?